src/hooks/useTvl.ts

```diff
@@ -30,9 +30,10 @@ export function useTvl(api: any) {
       };
 
       const priceUsd = async (): Promise<number> => {
-        if (defaultUnitToken.value === 'SDN') {
+        const chainName = await apiRef.runtimeVersion.specName.toString();
```

Comment:
This fetch is used across multiple scripts. Since the chain information does not change that frequently, I think we should optimize this by fetching the information only once and using that.
The same fetch is called:
- https://github.com/PlasmNetwork/astar-apps/blob/6656893b3422a86139c689be6782f553d1ff137f/src/hooks/useChainInfo.ts#L23
- https://github.com/PlasmNetwork/astar-apps/blob/1101eee8bc182fd2046852613590501a2beec85b/src/hooks/useApr.ts#L60
- https://github.com/PlasmNetwork/astar-apps/blob/6656893b3422a86139c689be6782f553d1ff137f/src/hooks/useMetaExtensions.ts#L79

It would be great if we only have to call the context `useChainInfo`

Reply:
It seems like `useChainInfo` is not working properly (returns `undefined`). I'll take a look and try to fix it.